Makefile

```diff
@@ -359,10 +366,12 @@ install-knative-eventing: | $(KUBECONFIG) $(KUBECTL) ## Install Knative Eventing
 
 .PHONY: install
 install: | $(KUBECTL) $(KO) install-knative-eventing install-rabbitmq-cluster-operator install-rabbitmq-topology-operator ## Install local dev Knative Eventing RabbitMQ - manages all dependencies, including K8S components
-	$(KO) apply --filename config/broker
+	$(KO) apply --local --platform=linux/amd64,linux/arm64 --filename config/broker && \
+    ./test/copy_images.sh
 	$(KUBECTL) wait --for=condition=available deploy/rabbitmq-broker-controller --timeout=60s --namespace $(EVENTING_NAMESPACE)
 	$(KUBECTL) wait --for=condition=available deploy/rabbitmq-broker-webhook --timeout=60s --namespace $(EVENTING_NAMESPACE)
-	$(KO) apply --filename config/source
+	$(KO) apply --local --platform=linux/amd64,linux/arm64 --filename config/source && \
```

Comment:
Not sure if we need this, ko apply already does what we need, sure this does not work on m1 macs without this line?